ert/serialization/_serializer.py

```diff
@@ -87,10 +87,14 @@ async def encode_to_path(
 
     async def decode_from_path(
         self, path: Union[str, Path], *args: Any, **kwargs: Any
-    ) -> Any:
+    ) -> Dict[str, float]:
```

Comment:
Is it ok that the base class returns `Any`?

Reply:
No complaints from `mypy` at least on this covariant return type.

Reply:
The logic I used was that the serialiser interface could not attempt to restrict the return type at all. A serialiser could very well return an object or a dict, right?